Packages/Sites/Neos.NeosIo/Resources/Private/TypoScript/Root.ts2

```diff
@@ -70,6 +56,33 @@ prototype(Neos.NeosIo:DefaultPage) < prototype(Page) {
 	@cache.entryTags.site = ${'Node_' + site.identifier}
 }
 
+prototype(Neos.NeosIo:DefaultPage.SectionTitle) < prototype(TYPO3.TypoScript:Value) {
```

Comment:
Would be good to move these TS objects into separate files to encourage reusability or this file will grow...